Phindr3D-Python/src/Data/Metadata.py

```diff
@@ -86,6 +91,43 @@ def metadataFileExists(self, omf):
             return False
         # else
 
+    def loadMetadataFile(self):
+        filepath = self.metadataFilename
+        metadata = read_table(filepath, usecols=lambda c: not c.startswith('Unnamed:'), delimiter='\t')
+        numrows = metadata.shape[0]
+        rows = []
+        # takes input metadata and stores in a list of tuples of each row
+        for i in range(numrows):
+            c1 = metadata.at[i, 'Channel_1']
+            c2 = metadata.at[i, 'Channel_2']
+            c3 = metadata.at[i, 'Channel_3']
+            well = metadata.at[i, 'Well']
+            field = metadata.at[i, 'Field']
+            stack = metadata.at[i, 'Stack']
+            metadatafile = metadata.at[i, 'MetadataFile']
```

Comment:
Make sure you know what the value of these variables will be if there is no such column name as "Well" or "Field", and what values will be in the dictionary as a result.
Also, what happens if Channel_2 and/or Channel_3 don't exist? Include error handling for the case of no channels, no stack, and/or no image id.